<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QLabel" name="portrait_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="warn_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
